### PR TITLE
Add autofix to block-opening-brace-space-before

### PR DESCRIPTION
**Pull request by superdem**
**Description**
Enable auto fix option for `block-opening-brace-space-before`

### CODE REVIEW

**Comment by jeddy3** [comment]
@superdem Thank you for starting on this PR!

I believe you'll need to add a `fixed` property to all of the tests for this rule, [like so](https://github.com/stylelint/stylelint/blob/master/lib/rules/at-rule-name-case/__tests__/index.js).

We can then be confident the code works.

**Comment by superdem** [comment]
Hi @jeddy3 , 
I've just updated the test with fixed. Hope it works now.

Thanks!

File: lib/rules/block-opening-brace-space-before/__tests__/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const { messages, ruleName } = rule;
 testRule(rule, {
   ruleName,
   config: ["always"],
-
+  fixed: true,
```
Comment (hudochenkov) [inline lib/rules/block-opening-brace-space-before/__tests__/index.js]: It should be `fix: true`. Please change it here and in all other places.

**Comment by hudochenkov** [comment]
Superseded by https://github.com/stylelint/stylelint/pull/3438. Sorry this PR wasn't ready.

**Pull request closed without merging**